### ISSUE TITLE
deepspeech/logits:0

### ISSUE THREAD

**Issue by 1flyingpig** [Fictionarry/ER-NeRF#75]
when i run deepspeech_features.py,i have an error
keyerror:"The name 'deepspeech/logits:0' refers to a Tensor which does not exist. The operation, 'deepspeech/logits', does not exist in the graph."

how to slove?
